terraform/main.tf

```diff
@@ -0,0 +1,82 @@
+data "aws_caller_identity" "current" {}
+
+resource "aws_dynamodb_table" "dimsio" {
+  name           = "${var.dynamodb_table_name}"
+  read_capacity  = 1
```

Comment:
Is potential throttling a concern with determining read & write capacities? Would enabling auto-scaling throughput capacity be a feasible option to cater for a more hands-off approach if this service were to be hit with throttling issues?